os/include/tinyara/pm/pm.h

```diff
@@ -307,20 +305,25 @@ enum pm_timer_type_e {
 	/* Scope for future expansion, up to 8 timer types can be supported */
 };
 
-/* This structure is used to send data to pm driver from app side */
-struct pm_arg_s {
-	enum pm_domain_e domain;
-	enum pm_state_e state;
+/* This structure is used to send data to pm driver from app side for timedSuspend */
+struct pm_suspend_arg_s {
+	enum pm_state_e state;            /* state to be suspended */
+	unsigned int timer_interval;      /* duration to be suspended */
 };
 
-typedef struct pm_arg_s pm_arg_t;
+typedef struct pm_suspend_arg_s pm_suspend_arg_t;
 
-struct pm_timer_s {
-	uint8_t timer_type;		/* Bits here are set according to the timer that is to be used */
-	uint32_t timer_interval;	/* The interval for whichever timer is to be used */
-	clock_t last_wifi_alive_send_time;       /* Time tick of the last wifi keep alive signal sent time */
+struct pm_wakeup_timer_s {	
```

Comment:
We use pm_timer in all APIs, But only in this structure data type, we call it pm_wakeup_timer.

Please change it to pm_timer as well.

Comment that our pm_timer means waking up the main core on time.